docs/plugin-proposal-regexp-modifiers.md

```diff
@@ -0,0 +1,81 @@
+---
+id: babel-plugin-proposal-regexp-modifiers
+title: "@babel/plugin-proposal-regexp-modifiers"
+sidebar_label: regexp-modifiers
+---
+
+## Example
+
+### `i` modifier
+```js title="input.js"
+// matches Aa and aa
+const regex = /(?i:a)a/
+```
+will be transformed to
+```js title="output.js"
+const regex = /(?:[Aa])a/
+```
+
+### `m` modifier
+```js title="input.js"
+// matches aa, a\naa, etc. but not a\na
+const regex = /(?m:^a)a/
+```
+will be transformed to
+```js title="output.js"
+const regex = /(?:(?:^|(?<=[\n\r\u2028\u2029]))a)a/
+```
+
+### `s` modifier
+```js title="input.js"
+// matches \na and aa, but not \n\n
+const regex = /(?s:.)./
+```
+will be transformed to
+```js title="output.js"
+const regex = /(?:[\s\S])./;
+```
+
+### multiple modifiers
+You can also apply multiple modifiers:
+
+```js
+// matches Aa, aa, A\naa, etc. but not A\na
+const regex = /(?im:^a)a/
+```
+
+This plugin only supports `i`, `m` and `s` modifier.
```

Comment:
```suggestion
This proposal only supports `i`, `m` and `s` as inline modifiers.
```

Reply:
The modifiers proposal also mentions the `x` flag, though it is covered in another proposal: https://github.com/tc39/proposal-regexp-x-mode. If we ever support that proposal, we should give a note here that to use the `x` modifier you should also install `@babel/plugin-proposal-regexp-x-mode`.